assembly/src/parsers/blocks.rs

```diff
@@ -291,3 +301,26 @@ pub fn combine_blocks(mut blocks: Vec<CodeBlock>) -> CodeBlock {
 
     blocks.remove(0)
 }
+
+/// Returns a CodeBlock [Span] from sequence of Span blocks provided as input.
+pub fn combine_spans(spans: &mut Vec<CodeBlock>) -> CodeBlock {
+    if spans.len() == 1 {
+        return spans.remove(0);
+    }
+
+    let mut ops = Vec::<Operation>::new();
+    for block in (*spans).iter() {
+        if let CodeBlock::Span(span) = block {
+            for batch in span.op_batches() {
+                ops.extend_from_slice(batch.ops());
+            }
```

Comment:
Nice! This is better than using `Span::get_ops()`.

Reply:
Thanks!